flow/raster_cache.cc

```diff
@@ -511,10 +511,7 @@ void RasterCache::SweepOneCacheAfterFrame(RasterCacheKey::Map<Entry>& cache,
 void RasterCache::CleanupAfterFrame() {
   picture_metrics_ = {};
   layer_metrics_ = {};
-  {
-    TRACE_EVENT0("flutter", "RasterCache::SweepCaches");
```

Comment:
This is almost always very fast and I've never used it in tracing. @flar can you comment on whehter you think this is worth keeping?